### PR TITLE
Update profile 

### PR DESCRIPTION
**Pull request by MadhuSaini22**
## Fixes Issue
Updated my profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2700"></a>

### CODE REVIEW

**Review by Sativarsainath-26 — COMMENTED** [review]
Looks good :thumbsup:  But you also added public/data/MadhuSaini22.json file which is not needed. Could you please revert those changes and only commit the file present in data/MadhuSaini22.json .

File: public/data/MadhuSaini22.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (eddiejaoude) [inline public/data/MadhuSaini22.json:1]: This file isn't needed

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: public/data/MadhuSaini22.json (hunk 1 above)
Comment (MadhuSaini22) [reply public/data/MadhuSaini22.json]: Thanks Eddie, you're too kind
We love you!!
done with changes now

**Pull request merged**